nss-cohort-36/bangazon-workforce-mgt-lords-of-might-magic
# Sp assigning training

# Description

you can now assign trainings to an employee

Closes #[Enter issue #]

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout sp-assigning-training`
4. go to an employees details and try adding some trainings

# Checklist:
